docs/get_started.rst

```diff
@@ -1,17 +1,369 @@
+
+    $ python -m pip install --user pipx
+
+.. image:: images/pipx.png
+
+To permanently modify the default environment variables:
+
+1. Click Start and search for "edit environment variables", or open System properties and click Advanced system settings.
+2. Click the Environment Variables button.
+3. To change System variables, you need non-restricted access to your machine (i.e. Administrator rights). Add the following paths to your PATH environment variable:
+
+   * ``%USERPROFILE%\.local\bin``
+   * ``%USERPROFILE%\AppData\Roaming\Python\Python38\Scripts``
+
+.. image:: images/env-var2.png
+
+.. note::  Be precise when entering these paths. Add them at the very end of the Variable Value already in place. Separate each path by a semicolon (;) with no space between path names.
+
+Open a new command prompt and verify that ``pipx`` is available::
+
+    pipx --version
+
+You should see a version number after entering this command, such as: ``0.12.3.1``.
+
+If you get an error instead, such as ``'pipx' is not recognized as an internal or external command,
+operable program or batch file.``, confirm that your environment variables have been updated.
+
+Finally, install CumulusCI:
+
+Inside your terminal application, enter the following command::
+
+    pipx install cumulusci
+
+Once finished `verify your installation`_.
+
+
+Verify Your Installation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In a new terminal window, verify that CumulusCI installed correctly by running ``cci version``.
+
+.. code-block:: console
+
+    $ cci version
+    CumulusCI version: 3.20.0 (/path/to/bin/cci)
+    Python version: 3.8.5 (/path/to/bin/python)
+
+    You have the latest version of CumulusCI.
+
+You can also use this command to check whether your CumulusCI installation is up to date.
+
+Still need help? Feel free to submit a question on our `Trailblazer community group <https://trailblazers.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009M9ZCAU>`_.
+
+
+
+Setup SFDX 
+----------
+
+While it's possible to use ``cci org connect <org_name>`` to connect to a persistent org, we recommend working with scratch orgs created by Salesforce DX.
+
+If you haven't set up Salesforce DX, follow these steps:
+
+1. `Install Salesforce CLI <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm>`_
+2. `Enable Dev Hub Features in Your Org <https://developer.salesforce.com/docs/atlas.en-us.228.0.sfdx_dev.meta/sfdx_dev/sfdx_setup_enable_devhub.htm>`_
+3. `Connect SFDX to Your Dev Hub Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm>`_ 
+    .. important:: Be sure to use the ``--setdefaultdevhubusername`` option!
+
+If you have the ``sfdx`` command installed, are connected to your devhub, and set the ``defaultdevhubusername`` config setting (use the ``sfdx force:config:list`` command to verify!), you're now ready to use ``cci`` with ``sfdx``.
```

Comment:
This seems really important. Should we elaborate more?

Reply:
We could link a second time to the Trailhead setup module (linked above).

Reply:
@davidmreed: Does the Trailhead elaborate further why cci works best with orgs created SFDX? Should we elaborate on this? I think that's what I was getting at.

Reply:
I think that's a pretty big conceptual topic ("Why use scratch orgs?") that we kind of tempt with the first sentence of this otherwise pragmatic section. I wonder if the introductory sentence should just be revised.

Reply:
@Br4nd0R Thoughts? If it's fine as is, I have no problem resolving convo. Just wanted to make sure we're not leaving any necessary details out.

Reply:
If I'm interpreting correctly, perhaps reword to de-emphasize the recommendation piece? "We assume you're using ``cci org connect <org_name>`` to connect to a scratch org created by Salesforce DX, which is more common than connecting to a persistent org."

Reply:
See, I feel like we're hedging instead of firmly directing. "While it's possible, we recommend." But _why_ do we recommend? Why not just drive them straight to it?

Reply:
As a model-isv, Salesforce.org sets the standard for on-platform development processes. Persistent orgs (sandboxes) have state drift over time. Multiple dev working on multiple sandboxes eventually become out of sync, tests become flakey, and people begin to lose confidence that what was developed in a sandbox will work the same in a production environment. @davidmreed has coined this "The Org Problem".

Scratch orgs (along with the CumulusCI flow; the workflow we recommend using which has a section unto itself elsewhere in the docs) allow for things to be done in a repeatable fashion across all members of a development team. With a persistent org, "starting fresh" (the equivalent of spinning up a new scratch org, installing a package and its dependencies along with any unpackaged metadata) is much more complicated.

I'm ok with just removing mention of the ability to connect to persistent orgs. We specific docs on how to connect persistent orgs which people can find if this is something they want to pursue. We just won't explicitly layout how to use the "CumulusCI Flow" with persistent orgs in our docs.